doc/dev/development.md

```diff
@@ -14,6 +14,7 @@ branch, but release branches of Broker should not change.
   - [Setting up Go](#setting-up-go)
   - [Setting up Bazel](#setting-up-bazel)
   - [Setting up Docker](#setting-up-docker)
+  - [Setting up personal access token](#setting-up-personal-access-token)
```

Comment:
Once this is in here, please make the same change to the mixer's development guide.